### PR TITLE
Poison Mushroom and P Acorn inside the Custom Powerup Block

### PR DESCRIPTION
**Pull request by poec987**
Adds the Poison Mushroom and P Acorn as options for the Custom Powerup Block

There is also a setting to change the block to be a brick block if you don't like those BOOORING question blocks.

### CODE REVIEW

File: source/actors/custompowerupblock.cpp [hunk 1]
```diff
@@ -31,7 +31,12 @@ u32 CustomPowerupBlock::onCreate() {
         return 2;
     }
 
-    this->tileId = 49; // Question Block
+    switch (eventID1 & 0xF) {
```
Comment (Luminyx1) [inline source/actors/custompowerupblock.cpp:34]: Add a comment for what nybble this is

File: source/actors/pacorn.cpp [hunk 2]
```diff
@@ -58,23 +58,36 @@ void PAcorn::vf18C() {
 }
 
 void PAcorn::playBlockReleaseUpAnim() {
-    this->model->playSklAnim("out", 1);
+    this->model->playSklAnim("out", 0);
+    this->model->sklAnims[0]->frameCtrl.shouldLoop(false);
+    this->_1810 = 1;
```
Comment (Luminyx1) [inline source/actors/pacorn.cpp:63]: What is this->_1810?

File: source/actors/poisonmushroom.cpp [hunk 3]
```diff
@@ -40,7 +42,7 @@ PoisonMushroom::PoisonMushroom(const ActorBuildInfo* buildInfo)
 u32 PoisonMushroom::onCreate() {
     this->_1827 = true;
     this->_1808 = 0;
-    this->model = ModelWrapper::create("I_kinxkx", "I_kinxkx");
+    this->model = ModelWrapper::create("I_kinxkx", "I_kinxkx", 5); // Funny
```
Comment (Luminyx1) [inline source/actors/poisonmushroom.cpp:45]: huh

**Review by Luminyx1 — APPROVED** [review]

File: source/actors/pacorn.cpp (hunk 2 above)
Comment (poec987) [reply source/actors/pacorn.cpp:63]: I am not exactly sure, but I found it in the hammer flower code and it works so I am not complaining. Probably animation id thingy, but idk.

**Pull request merged**